chronosphere/tfschema/notification_policy.go

```diff
@@ -71,23 +69,27 @@ var NotificationPolicy = map[string]*schema.Schema{
 	},
 }
 
-var NotificationRouteSchema = typeset.Set{
-	ElemFields: map[string]typeset.ElemField{
-		// Note, severity is case-sensitive.
-		"severity": typeset.NotNormalized(&schema.Schema{
-			Type:     schema.TypeString,
-			Required: true,
-		}),
-		"notifiers": typeset.NotNormalized(&schema.Schema{
-			Type:     schema.TypeSet,
-			Optional: true,
-			Elem:     &schema.Schema{Type: schema.TypeString},
-		}),
-		"repeat_interval": Duration{
-			Optional: true,
+var NotificationRouteSchema = &schema.Schema{
+	Type:     schema.TypeList,
```

Comment:
The set provided better diffs when there's a lot of routes, but with duplicate routes for the same severity blocked, there'll only be a warn + critical route, so the set complexity doesn't seem worthwhile.

This caught an issue during testing, where a test with 2 routes with the same severity, that referenced notifiers that were to-be-created. In `CustomizeDiff`, we'd see routes with `notifiers = []` (since the ID isn't known yet), but then both routes would look the same (regardless of notifiers referenced), and be de-duplicated into 1. So the plan would not detect the duplicate route. Using a list avoids this (and other set-related bugs in terraform).